custom_components/zwave_mqtt/light.py

```diff
@@ -0,0 +1,125 @@
+"""Support for Z-Wave lights."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_TRANSITION,
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_TRANSITION,
+    Light,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .const import DOMAIN
+from .entity import ZWaveDeviceEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up Z-Wave Light from Config Entry."""
+
+    @callback
+    def async_add_light(values):
+        """Add Z-Wave Light."""
+        light = ZwaveDimmer(values)
+        async_add_entities([light])
+
+    async_dispatcher_connect(hass, "zwave_new_light", async_add_light)
+
+    await hass.data[DOMAIN][config_entry.entry_id]["mark_platform_loaded"]("light")
+
+    return True
```

Comment:
Drop this. It's not necessary.

Reply:
It's in line with the switch component. What does it do ?

Reply:
It's been long gone 😉 #40

It doesn't do anything.